spec/spec_helper.rb

```diff
@@ -26,9 +28,27 @@
     rspec.syntax = :expect
   end
 
+  if ENV.fetch("OPENAI_ACCESS_TOKEN", nil)
+    warning = "WARNING!
+    Specs are hitting the OpenAI API using your OPENAI_ACCESS_TOKEN!
+    This costs at least 2 cents per run and is very slow!
+    If you don't want this, unset OPENAI_ACCESS_TOKEN to just run against the stored VCR responses.
```

Comment:
seems to me `ENV["NO_VCR"] == "true"` is the better way to toggle "please run live?"

Reply:
Decided to try with OPENAI_ACCESS_TOKEN based on a couple things

- Simpler - just provide a token, and the tests will use it, even if a dev doesn't know what VCR is
- OpenAI change the API in a breaking way quite often, I want to run the specs vs the API more often in order to catch this quicker

Downsides

- Risk of costing money unexpectedly (mitigating with warnings)

If it causes problems I'll rethink.